packages/cf-builder-card/src/CardBuilder.js

```diff
@@ -48,21 +66,7 @@ class CardBuilder extends React.Component {
   }
 }
 
-CardBuilder.propTypes = {
```

Comment:
Right now the alternative is either:

```js
type Props = {
  propName: string;
};

type DefaultProps = {
  propName: string
};

type State = {
  stateName: string
};

class MyComponent extends React.Component<DefaultProps, Props, State> {
  constructor(props, context) {
    super(props, context);
    this.state = {
      stateName: this.props.propName;
    };
  }
  render() {
    return <div/>
  }
}

MyComponent.propTypes = {
  propName: React.PropTypes.string.isRequired
};

MyComponent.defaultProps = {
  propName: 'foo'
};
```

Or:

```js
class MyComponent extends React.Component<*, *, *> {
  // ...
}

MyComponent.propTypes = {
  propName: React.PropTypes.string.isRequired
};

MyComponent.defaultProps = {
  propName: 'foo'
};
```

> ^ Please do not do this second option, `*` is magic that does not work the way you expect it to.

In the future when you are using flow everywhere you can switch to this:

```js
type Props = {
  propName: string;
};

class MyComponent extends React.Component {
  props: Props;
  
  static defaultProps = {
    propName: 'foo'
  };

  state = {
    stateName: this.props.propName;
  };

  render() {
    return <div>
  }
}
```